### PR TITLE
two column - Alvin Cho

### PR DESCRIPTION
**Pull request by c-alvin**
image

### CODE REVIEW

File: css-two-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+          <div class="column-three-fourth">
+            <h2 class="fontkarla400">DATA THAT TRANSFORMS</h2>
+            <p class="fontkarla300 fontweight100">We understand your facility better than
+            you do; we provide visually-appealing
+            eal-time analytics, reports and trends
+            at your finger tips.</p>
+          </div>
+        </div>
+      </div>
+    </div>
```
Comment (codingcodymiller) [inline css-two-column-layout/index.html:77]: Your current setup is this:
```
row
  column-half x2
    row flex-wrap
      column-fourth
      column-three-fourth
      column-fourth
      column-three-fourth
```

Instead, I recommend this:

```
row flex-wrap
  column-half x4
    row
      column-fourth
      column-three-fourth
```

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by c-alvin** [comment]
image

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Your sections are in the incorrect order 👀 
image

**Comment by c-alvin** [comment]
image

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**